hoot-core/src/main/cpp/hoot/core/io/ApiDbReader.h

```diff
@@ -51,6 +51,8 @@ class ApiDbReader
 {
 public:
 
+  static QString emailKey() { return "api.db.email"; }
```

Comment:
This function shouldn't be necessary, wouldn't you want to use `ConfigOptions::getApiDbEmailKey()` which is static?  Or if you absolutely need this function, you should call that function within it.